voluptuous/tests/tests.py

```diff
@@ -8,6 +8,8 @@
     validate, ExactSequence, Equal, Unordered
 )
 from voluptuous.humanize import humanize_error
+from voluptuous.encoding import to_utf8_py2
+import six
```

Comment:
I've had to make separate tests for py2 & 3 due to checking for unicode or str objects.